network/network.go

```diff
@@ -244,7 +244,7 @@ func NewNetwork(
 	// Track all default bootstrappers to ensure their current IPs are gossiped
 	// like validator IPs.
 	for _, bootstrapper := range genesis.GetBootstrappers(config.NetworkID) {
-		ipTracker.ManuallyTrack(bootstrapper.ID)
+		ipTracker.ManuallyGossip(bootstrapper.ID)
```

Comment:
From the PR description, I would have expected this to use `ManuallyTrack` instead of `ManuallyGossip` (after dividing into two separate functions). Why do we need to gossip the bootstrappers here?

Reply:
These are not custom bootstrappers - these are the default bootstrappers. See: https://github.com/ava-labs/avalanchego/tree/master/network#peerlist

Reply:
The reason we gossip these nodeIDs is to more gracefully handle IP changes of bootstrappers.